### PR TITLE
Update LFG. Changed link to button link

### PR DESCRIPTION
**Pull request by Waff3n**
Changed link invite to channel from description to button

Added Invites.cs to handle invites in memory

Updated DSharpPlus, DSharpPlus.CommandsNext, DSharpPlus.Interactivity to 4.4.0

In DiscordConfiguration set LogUnknownEvents to false, because discord have added new event which not supported by 4.4.0

Updated MaxMind.GeoIP2 to 5.1.0

Updated MySql.Data to 8.0.33

### CODE REVIEW

File: Bot_NetCore/Bot.cs [hunk 1]
```diff
@@ -19,6 +19,7 @@
 using DSharpPlus.Interactivity;
 using DSharpPlus.Interactivity.Enums;
 using DSharpPlus.Interactivity.Extensions;
+using Google.Protobuf;
```
Comment (Vanguard-Fx) [inline Bot_NetCore/Bot.cs:22]: Using нигде не используется

File: Bot_NetCore/Listeners/VoiceListener.cs [hunk 2]
```diff
@@ -477,14 +477,11 @@ public static async Task UpdateFindChannelEmbedOnVoiceStateUpdated(DiscordClient
                                     }
                                     else
                                     {
-                                        content += $"{DiscordEmoji.FromName(client, ":gold:")} ☐\n";
+                                        content += $"{DiscordEmoji.FromName(client, ":warning:")} ☐\n";
```
Comment (Vanguard-Fx) [inline Bot_NetCore/Listeners/VoiceListener.cs:480]: Я бы вернул :doubloon: и :gold:

File: Bot_NetCore/Misc/Invites.cs [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+    {
+        if (LfgInvites.TryGetValue(channel.Id, out var channelInvite))
+        {
+            return channelInvite;
+        }
+
+        var invite = await channel.CreateInviteAsync(max_age: 0);
+        return invite.ToString();
+    }
+}
```
Comment (Vanguard-Fx) [inline Bot_NetCore/Misc/Invites.cs]: Этот класс вообще нужен?

**Review by Vanguard-Fx — CHANGES_REQUESTED** [review]

File: Bot_NetCore/Listeners/VoiceListener.cs (hunk 2 above)
Comment (Waff3n) [reply Bot_NetCore/Listeners/VoiceListener.cs:480]: Делал для теста, проморгал, вернем

File: Bot_NetCore/Misc/Invites.cs (hunk 3 above)
Comment (Vanguard-Fx) [reply Bot_NetCore/Misc/Invites.cs]: Нужен

**Review by Vanguard-Fx — APPROVED** [review]

**Pull request merged**